src/components/shared/ButtonWithPanel/index.js

```diff
@@ -169,15 +169,16 @@ export class ButtonWithPanel extends React.PureComponent<Props, State> {
         className={classNames('buttonWithPanel', className, { open })}
         ref={this._wrapperRef}
       >
-        <input
+        <button
```

Comment:
This is so that we can add `::before` and `::after` on them. These pseudo-selectors add another inline element _inside_ the target element.
See https://codepen.io/julienw/pen/dyXrLGg for example.